notebook/services/edit.html

```diff
@@ -34,7 +34,7 @@
     <div id="menus" class="navbar navbar-default" role="navigation">
```

Comment:
I moved it back where it belongs, and regenerated i18n/nbui.pot so that it picks up the strings appropriately.